task/rhacs-image-scan/3.71/rhacs-image-scan.yaml

```diff
@@ -49,28 +52,119 @@ spec:
 
         When set to "true", skip verifying the TLS certs of the Central endpoint and registry.
       default: "false"
+  volumes:
+    - name: token-vol
+      projected:
+        sources:
+          - serviceAccountToken:
+              audience: api
```

Comment:
Let's use a different audience other than `api`, since `api` is for the default API server.

`rox` or `rhacs` should suffice.